worldedit-forge/src/main/resources/mcmod.info

```diff
@@ -14,7 +14,8 @@
         "Forge@[${forgeVersion},)"
     ],
     "dependencies": [
-        "Forge@[${forgeVersion},)"
+        "Forge@[${forgeVersion},)",
+        "ForgeMultipart"
```

Comment:
No, the server will not crash. This is a soft-depend, `requiredMods` is for hard.